### PR TITLE
Add Shaarli and Pocket HTML imports

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes, idea from @nhoizey
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | **no** 
| Documentation | no
| Translation   | yes
| CHANGELOG.md  | no
| License       | MIT

~~Still one failing test.~~ 

Fix https://github.com/wallabag/wallabag/issues/6776

### CODE REVIEW

**Comment by nhoizey** [comment]
@nicosomb a few screenshots of the steps to export Shaarli data:

In the "Tools" section:

image

Click on "Export database":

image

Select if you want to export "all", "public" only, or "private" only bookmarks, and if you want links in the export to target Shaarli items or the actual bookmarked URLs, and click the "Export" button:

image

Save your old fashioned, Firefox inspired, HTML:

image

File: src/Wallabag/ImportBundle/Controller/ImportController.php
Comment (j0k3r) [inline src/Wallabag/ImportBundle/Controller/ImportController.php:1]: Don't you need to add `pocket_html` here too?

**Review by j0k3r — COMMENTED** [review]
Rebase needed

**Review by j0k3r — APPROVED** [review]

**Comment by bmp** [comment]
Hello, is this part of 2.6.5 release? I have Shaarli and the export as indicated in the images is a HTML file, but I am currently unable to import the HTML file on wallabag (I am on 2.5.4 with yunohost currently).

**Comment by bmp** [comment]
Thanks a tonne for the clarification @nicosomb!

**Pull request merged**